dataland-frontend/src/assets/scss/theme.scss

```diff
@@ -0,0 +1,181 @@
+@import "variables";
+
+:root {
+  // BRAND COLORS
+  --text-color: #{$brown-dark};
+  --text-color-secondary: #{$brown};
+  --text-color-third: #{$brown-light};
+  --primary-color: #{$orange-prime};
+  --primary-color-text: #{$paper-white};
+  --secondary-color: #{$aquamarine-dark};
+  --secondary-color-text: #{$paper-white};
+
+  // added colors
+  --primary-color-100: #FDEFE6;
+  --color-blue: #{$aquamarine-dark};
+
+  // default prime colors
+  --surface-0: #ffffff;
+  --surface-50: #FAFAFA;
+  --surface-100: #F5F5F5;
+  --surface-200: #EEEEEE;
```

Comment:
Are those colors the defaults of PrimeVue? If yes - we can remove the re-declaration here.

Reply:
They should be declared for the prime color profile that we used. Based on them, prime generates classes like e.g. .text-purple-300 , which we can use in projections.

Reply:
ok